data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/EncryptionConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EncryptionConfig {
+    @JsonProperty("type")
+    private EncryptionType type = EncryptionType.SSL;
```

Comment:
Does this deserialization work currently? You don't have to change but just curious. See the bug context here (https://github.com/opensearch-project/data-prepper/issues/3376)

Reply:
It seemed to be working in my testing.

Reply:
Yeah okay you're right these aren't working any more.